local-tests/setup/shiva-client.ts

```diff
@@ -152,10 +168,15 @@ export class TestnetClient {
       .then((res: Response) => {
         if (res.status === 200) {
           return res.json();
+        } else {
+          throw res; // throw as an error to the caller
```

Comment:
Can we make this throw an error with `res.text()` as it's message instead of throwing the entire Response object?